pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -1741,9 +1741,20 @@ func testRedisProtocolClassificationInner(t *testing.T, tr *Tracer, clientHost,
 				extras:        make(map[string]interface{}),
 			},
 			postTracerSetup: func(t *testing.T, ctx testContext) {
+				dialContext := defaultDialer.DialContext
+				if withTLS {
+					tlsDialer := &tls.Dialer{
+						NetDialer: defaultDialer,
+						Config: &tls.Config{
+							InsecureSkipVerify: true,
+						},
+					}
+					dialContext = tlsDialer.DialContext
+				}
```

Comment:
why only for this test?

Reply:
The Redis client wrapper we use in the other tests already does this.

Reply:
let's add an inline comment in the code
that test is not using the client to create an actual protocol error, therefore we need ....